### PR TITLE
added cookiecutter, pre-commit, terraform-docs, tflint, tfsec

### PR DESCRIPTION
**Pull request by aidanmelen**
added the following whalebrew packages:

- cookiecutter
- pre-commit
- terraform-docs
- tflint
- tfsec

### CODE REVIEW

**Review by tjamet — CHANGES_REQUESTED** [review]
Thanks for the awesome contribution!
Can we manage to reduce the number of supported versions?

File: cookiecutter/tags.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+versions:
```
[the remark this answers is not in the document]
Comment (aidanmelen) [reply cookiecutter/tags.yaml:1]: I can slim down the list.

File: tflint/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ARG VERSION=latest
+FROM wata727/tflint:${VERSION}
```
[the remark this answers is not in the document]
Comment (aidanmelen) [reply tflint/Dockerfile:2]: > Is this the official tflint image?

Yes, [source for github repo](https://github.com/terraform-linters/tflint#docker)

File: tfsec/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+ARG VERSION=latest
+FROM liamg/tfsec:${VERSION}
```
[the remark this answers is not in the document]
Comment (aidanmelen) [reply tfsec/Dockerfile:2]: Yes, [source from github repo](https://github.com/tfsec/tfsec#use-with-docker)

**Review by tjamet — APPROVED** [review]

**Pull request merged**